### ISSUE TITLE
When proposing class, I should be warned when the resource with the same label exists

### ISSUE THREAD

**Issue by tomas-knap** [odalic/sti#382]
When proposing class, I may easily propose the same resource twice. I only need to click "OK" twice. 

Solution: It should warn me when such the resources with the same label already exists in the knowledge base.

**Comment by brodecva** [comment]
@tomas-knap Hi, in the current state we only check the URIs, so naturally when the users leave them to generate automatically, a unique URI will be generated even for two resources sharing the same label.

**Comment by tomas-knap** [comment]
Yes, for now it is OK, leave it as it is. But it would be good for the user to give him a warning that such label is already used (because this is usually the case when the use does not search but directly proposes or when he proposes accidentally twice)

**Comment by brodecva** [comment]
@tomas-knap Oh, right, i did not notice the master-theses label. :-)
